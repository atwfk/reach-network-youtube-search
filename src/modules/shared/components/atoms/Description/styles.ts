import styled from "styled-components";
import { COLORS, SCREENS } from "../../../constants";

export const StyledDescription = styled.p`
  color: ${COLORS.GREY_SHD1};
  font-size: 15px;

  @media (min-width: ${SCREENS.MD}) {
    font-size: 20px;
  }
`;
