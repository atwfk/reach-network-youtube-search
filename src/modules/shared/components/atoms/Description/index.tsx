import React from "react";
import type { FC, ReactElement } from "react";
import { StyledDescription } from "./styles";

const Description: FC<{ description: string }> = ({ description }): ReactElement => {
  return <StyledDescription className="description">{description}</StyledDescription>;
};

export default Description;
