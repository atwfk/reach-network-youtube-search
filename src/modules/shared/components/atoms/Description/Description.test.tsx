import { cleanup } from "@testing-library/react";
import "jest-styled-components";
import renderer from "react-test-renderer";
import Description from ".";

afterEach(cleanup);

describe("Description Component snapshots", () => {
  it("Should render simple Description component", () => {
    const tree = renderer
      .create(
        <Description description="Full Kubernetes Tutorial | Kubernetes Course | Hands-on course with a lot of demos Become a Kubernetes Administrator - CKA: ..." />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
