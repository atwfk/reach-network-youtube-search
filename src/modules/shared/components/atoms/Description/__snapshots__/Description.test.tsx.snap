// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Description Component snapshots Should render simple Description component 1`] = `
.c0 {
  color: #767272;
  font-size: 15px;
}

@media (min-width:768px) {
  .c0 {
    font-size: 20px;
  }
}

<p
  className="c0 description"
>
  Full Kubernetes Tutorial | Kubernetes Course | Hands-on course with a lot of demos Become a Kubernetes Administrator - CKA: ...
</p>
`;
